server/command.go

```diff
@@ -199,14 +203,56 @@ func (p *Plugin) runHelpCommand(user *model.User) (string, error) {
 	if p.canConnect(user) {
 		text += "\n" + strings.ReplaceAll(oAuthHelpText, "|", "`")
 	}
+
+	text += "\n" + strings.ReplaceAll(followStatusHelpText, "$", "`")
+
 	return text, nil
 }
 
+func (p *Plugin) runEnableDisableFollowStatus(user *model.User, args *model.CommandArgs) (string, error) {
+	split := strings.Fields(args.Command)
+
+	if len(split) == 2 { // get and show status
+		enabled, appErr := p.getFollowStatusForUser(user.Id)
+		if appErr == nil {
+			var status string
+			if enabled {
+				status = "on"
+			} else {
+				status = "off"
+			}
+			return fmt.Sprintf("Your current `status_sync` setting is `%v`", status), nil
+		} else {
+			return "Your current `status_sync` setting is not set.", nil
+		}
+	}
+	if len(split) == 3 {
+		var err *model.AppError
+		var text string
+		switch split[2] {
+		case "on":
+			err = p.setFollowStatusForUser(user.Id, true)
+			text = "Automatically following your Zoom status."
+		case "off":
+			err = p.setFollowStatusForUser(user.Id, false)
+			text = "Not following your Zoom status."
+		default:
+			text = fmt.Sprintf("Invalid value `%v`. Accepted values: `on`, `off`.", split[2])
+		}
+		if err != nil {
+			return "Could not set your `status_sync` settings", err
+		}
+		return text, nil
+	} else {
+		return "Incorrect number of arguments. Usage: `/zoom status_sync [on|off]`", nil
+	}
+}
+
 // getAutocompleteData retrieves auto-complete data for the "/zoom" command
 func (p *Plugin) getAutocompleteData() *model.AutocompleteData {
 	available := "start, help"
 	if p.configuration.EnableOAuth && !p.configuration.AccountLevelApp {
-		available = "start, connect, disconnect, help"
+		available = "start, connect, disconnect, status_sync [ on | off ], help"
```

Comment:
Would we prefer to bury this under a settings command for possible future settings? (0/5, we could apply YAGNI and add the settings command when we need it)